rainbow-me/browser-extension
# feat: pin / unpin tokens

Fixes BX-1328
Figma link (if any):
- [Context menu design](https://www.figma.com/file/hzfwKOddfRmeVO4KUsG9wM/branch/aLkgDopprmzkQRc7M796PN/Rainbow-BX?type=design&node-id=16098-14055&mode=design&t=2dSNDGuOQ3lPuU5M-0)
- [Highlighter indicator design](https://www.figma.com/file/hzfwKOddfRmeVO4KUsG9wM/branch/aLkgDopprmzkQRc7M796PN/Rainbow-BX?type=design&node-id=16099-15553&mode=design&t=pCrl8f03u4whihbA-0)

## Screen recordings / screenshots

https://github.com/rainbow-me/browser-extension/assets/53529533/cc5d837c-78d1-49d1-98de-61585579b401

## What to test
- Go to tokens tab on homepage
- Press the token row and make sure it pins at first and unpins if you press again
- When token is pinned it gets on top of the tokens list
- When right clicking you'll also see pin / unpin token option. Should work exactly the same, but without pressing.
